modules/ROOT/pages/java-embedded/setup.adoc

```diff
@@ -172,7 +172,6 @@ var initialMembers = List.of(
 );
 
 var managementService = new ClusterDatabaseManagementServiceBuilder( homeDirectory )
-    .setConfig( GraphDatabaseSettings.mode, CORE )
```

Comment:
Removal is fine. However there are some other changes we should do too:

- In the config file version of this on line 139
  - Remove `dbms.mode=CORE`
  - `dbms.default_advertised_address` -> `server.default_advertised_address`
  - `dbms.default_listen_address` -> `server.default_listen_address`
  - `causal_clustering.discovery_type` -> `dbms.cluster.discovery.type`
  - `causal_clustering.initial_discovery_members` -> `dbms.cluster.discovery.endpoints`
- Replace `CausalClusteringSettings.initial_discovery_members` with `DiscoverySettings.discovery_endpoints`